web/src/unread_ui.ts

```diff
@@ -64,7 +65,8 @@ export function reset_unread_banner(): void {
 
 export function notify_messages_remain_unread(): void {
     if (!user_closed_unread_banner) {
-        $("#mark_read_on_scroll_state_banner").toggleClass("invisible", false);
+        $("#mark_read_on_scroll_state_banner").toggleClass("hide", false);
+        $("#typing_notifications").css("margin-top", "-18px");
```

Comment:
This should be done with new rules in the CSS files -- we basically never add CSS via `.css()`.

Reply:
oh! I git grepped .css() and found some files using that, so I thought I could use it. I'll update it.

Reply:
Updated!